lib/list.g

```diff
@@ -881,6 +881,34 @@ DeclareSynonym( "IntersectBlist", INTER_BLIST );
 ##
 DeclareSynonym( "SubtractBlist", SUBTR_BLIST );
 
+#############################################################################
+##
+#F  MeetBlist( <blist1>, <blist2> )
+##
+##  <#GAPDoc Label="MeetBlist">
+##  <ManSection>
+##  <Func Name="MeetBlist" Arg='blist1, blist2'/>
+##
+##  <Description>
+##  Returns <K>true</K> if <A>blist1</A> and <A>blist2</A> have <K>true</K> in the same
+##  position and <K>false</K> otherwise. It is equivalent to, but faster than
```

Comment:
Here is a suggestion, if someone (esp. a native speaker) has a better suggestion I am happy 
```suggestion
##  Returns <K>true</K> if there is a position at which both <A>blist1</A>
##  and <A>blist2</A> contain <K>true</K>, and <K>false</K> otherwise.
##  It is equivalent to, but faster than
```